docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -339,7 +339,7 @@ The following example Access Control Lists (ACLs) illustrate common scenarios fo
 Access for a new service account in a Kafka instance::
 +
 --
-I’ve created a new service account and I want to allow it to create and delete topics in the instance, to produce and consume messages in any topic in the instance, and to use any consumer group and any producer.
+You’ve created a new service account and you want to allow it to create and delete topics in the instance, to produce and consume messages in any topic in the instance, and to use any consumer group and any producer.
```

Comment:
Yeah, I see what Stetson had in mind here with describing these example scenarios. But I think "you" works just as well and feels more in line with the other 99.99% of our content.